extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -200,23 +200,24 @@ export class VisibilityModel {
    * Function that visibilityManager can used to dispose model or reset model
```

Comment:
"or repeat model"

Reply:
Intentionally did not change this one, but not 100% sure that was correct